uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_contact-actions-menu.scss

```diff
@@ -0,0 +1,70 @@
+.crm_contact-actions {
```

Comment:
the file name does not match the name of the component

Reply:
fixed!